cmd/ack-generate/command/crossplane.go

```diff
@@ -42,16 +42,20 @@ func generateCrossplane(_ *cobra.Command, args []string) error {
 	if len(args) != 1 {
 		return fmt.Errorf("please specify the service alias for the AWS service API to generate")
 	}
+
+	// Set vars to pass onto loadModel so it uses crossplane specific
+	// configurations
+	cpAPIGroupSuffix = "aws.crossplane.io"
+	cpDefaultCfg = true
+
 	ctx, cancel := contextWithSigterm(context.Background())
 	defer cancel()
 	if err := ensureSDKRepo(ctx, optCacheDir, optRefreshCache); err != nil {
 		return err
 	}
 	svcAlias := strings.ToLower(args[0])
 	optGeneratorConfigPath = filepath.Join(optOutputPath, "apis", svcAlias, optGenVersion, "generator-config.yaml")
-	m, err := loadModelWithLatestAPIVersion(
-		svcAlias,
-	)
+	m, err := loadModel(svcAlias, optGenVersion)
```

Comment:
and then here you would do:

```
m, err := loadModel(svcAlias, optGenVersion, "aws.crossplane.io", cpconfig.DefaultConfig)
```

This would mean you would not have to import the `pkg/generate/config/crossplane` package within `cmd/ack-generate/command/common.go`.